msgcntr/messageforums-app/src/webapp/jsp/dfReviseTopicSettingsAttach.jsp

```diff
@@ -51,12 +51,16 @@
 		}
 
 		function updateGradeAssignment() {
+			const topicAssignments = document.getElementById("revise:topic_assignments");
+
+			if (topicAssignments?.value === undefined || topicAssignments.value === "Default_0") return;
 
 			const associations = document.querySelectorAll('sakai-rubric-association');
-			const topicAssignments = document.getElementById("revise:topic_assignments");
 			const createTaskGroup = document.getElementById("revise:createTaskGroup");
 			const createTaskEmptyPanel = document.getElementById("revise:createTaskEmptyPanel");
-			if (topicAssignments?.value && topicAssignments.value !== "Default_0") {
+			const isTasksWidgetAvailable = !!document.querySelector("sakai-tasks");
+
+			if (isTasksWidgetAvailable) {
```

Comment:
```suggestion
			if (!!document.querySelector("sakai-tasks")) {
```

Reply:
No need for a variable there, really. You may as well just compute the boolean in the if statement. I know I'm being picky :)